web-app/src/pages/Dashboard/container.js

```diff
@@ -33,109 +29,165 @@ const DashboardContainer = Wrapped =>
       const userId = localStorage.getItem('user_id');
       getUserProfile(userId)
         .then(response => {
-          this.setState({
```

Comment:
Do we need to do this? The user profile is stored in Redux

Reply:
For some reason the user doesn't exist in the redux store. When the page loads. not sure what the issue is